incubator/hoverfly/templates/route.yaml

```diff
@@ -0,0 +1,59 @@
+{{ if .Values.openshift.route.admin.enabled }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}" 
+  name: "{{ template "fullname" . }}-admin"
+spec:
+  host: "{{ .Values.openshift.route.admin.hostname }}"
+  port:
+    targetPort: admin
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: "{{ template "fullname" . }}" 
+    weight: 100
+status:
```

Comment:
I don't think the pod status object should be in the template, because it describes the current state of the resource. It's not a spec for creating these routes.

Reply:
otherwise LGTM

Reply:
@tommysitu This is actually due to an issue between Helm and Openshift (and how Helm is currently  validating the Openshift OpenAPI spec). You can see more of the discussion here: https://github.com/openshift/origin/issues/24060 , but the current workaround is to stub out a `status` object to get around this. Let me know how you'd like to proceed here.

Reply:
fair enough. we should remove it once that openapi spec is fixed

Reply:
Absolutely. I'm tracking this for a few other charts as well, so I'll make sure to circle back on this one as well.